sycl/test/basic_tests/accessor/accessor_implicit_conversion_readwrite_to_read.cpp

```diff
@@ -0,0 +1,47 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
is this only a test that it compiles? It seems like you want this to be run. If so, it'll probably have to go into the llvm-test-suite repo, because those tests are run with devices, whereas the in-tree lit tests are run without any.

Reply:
Thanks @cperkinsintel 
Makes sense, I move the code to llvm-test-suite to make sure it runs on device.